components/AllocatedWorkers/AddAllocatedWorker/AddAllocatedWorker.jsx

```diff
@@ -52,19 +56,23 @@ const AddAllocatedWorker = ({ personId, currentlyAllocated }) => {
     setIsModalOpen(false);
     setWorkers();
   });
-  const addWorker = useCallback(async (formData) => {
+  const addWorker = useCallback(async ({ worker }) => {
```

Comment:
Rather basic react question, never really used `useCallback`, what does it exactly do?, from the docs it seems to help with optimisation and avoiding unnecessary renders

Reply:
Every time something change in a component (i.e. state, useEffects) the whole component is rerun.
You use `useCallback` if you want to "store" the definition of the function, so you don't need to reexecute its definition on each rerun.
As you said is an extra optimisation, it really depends how many times you think that component might render. In this case, as the user can click on several sections it could happen quite a few times, so `useCallback` is a nice to have